integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -203,9 +203,53 @@ public static boolean supportsIndexClosingAndOpening() {
 		return isActualVersion(
 				// See https://docs.aws.amazon.com/opensearch-service/latest/developerguide/supported-operations.html#version_7_1
 				// See https://docs.aws.amazon.com/opensearch-service/latest/developerguide/supported-operations.html#version_7_4
-				esVersion -> !( esVersion.isAws() && esVersion.isLessThan( "7.4" ) ),
+				es -> !( es.isAws() && es.isLessThan( "7.4" ) ),
 				// See https://docs.aws.amazon.com/opensearch-service/latest/developerguide/supported-operations.html#version_opensearch_1.0
-				osVersion -> true
+				os -> true,
+				aoos -> false
```

Comment:
```suggestion
				aoss -> false
```
this one caught my eye 😃